test/campaigns.js

```diff
@@ -1,43 +1,189 @@
-const chai = require('chai')
-const expect = chai.expect
-
+const { expect } = require('chai')
+const nockHelper = require('./helpers/nock_helper')
 const Hubspot = require('..')
-const hubspot = new Hubspot({ apiKey: 'demo' })
+const hubspot = () =>
+  new Hubspot({
+    accessToken: process.env.ACCESS_TOKEN || 'fake-token',
+  })
+const appId = process.env.APP_ID || 23456
 
 describe('campaigns', function() {
   describe('get', function() {
+    beforeEach(
+      nockHelper.mockOauthEndpoint('/email/public/v1/campaigns', {
+        campaigns: [],
+      }),
+    )
+    afterEach(nockHelper.resetNock)
+
     it('Should return campaign IDs with recent activity associated with the portal', function() {
-      return hubspot.campaigns.get().then(data => {
-        expect(data.campaigns).to.be.an('array')
-        expect(data.hasMore).to.equal(true)
-      })
+      return hubspot()
+        .campaigns.get()
+        .then(data => {
+          expect(data.campaigns).to.be.an('array')
```

Comment:
Thoughts on keeping the `hasMore`in the expectation as it is helpful information that's included in the data?

Reply:
Unfortunately `hasMore` is true or false depending on how many campaigns are in the result (since the response is paginated). I removed it because using my api token changed this to false.